internal/pkg/agent/application/monitoring/v1_monitor.go

```diff
@@ -908,6 +910,92 @@ func (b *BeatsMonitor) injectMetricsInput(cfg map[string]interface{}, componentI
 		},
 	}
 
+	for _, comp := range componentList {
+		logp.L().Infof("input component %s: %#v", comp.ID, comp.InputSpec.Spec.Service)
```

Comment:
Using the global logger is not really recommened, I don't think this should be used in new code.

Reply:
ooops, debugging statement I kept in there...